conftest.py

```diff
@@ -70,16 +70,83 @@ def flask_client(app):
     return app.test_client()
 
 
+@pytest.fixture
+def cli_runner(app):
+    return app.test_cli_runner()
+
+
 @pytest.fixture
 def dummy_uid():
     return 'user123'
 
 
+@pytest.fixture
+def dummy_uids():
+    return ['user124', 'user125', 'user126']
+
+
 @pytest.fixture
 def dummy_participant_uid():
     return 'pig'
 
 
+@pytest.fixture
+def create_newdle(dummy_uid, db_session):
+    """Returns a callable which lets you create dummy newdles"""
+
+    def _create_newdle(id_=None, **kwargs):
+        kwargs.setdefault(
+            'participants',
+            {
+                Participant(
+                    code=f'part1{id_}' if id_ else 'part1',
+                    name='Guinea Pig',
+                    email='example@example.com',
+                    auth_uid='pig',
+                ),
+            },
+        )
+        kwargs.setdefault(
+            'timeslots',
+            [
+                datetime(2019, 9, 11, 13, 0),
+                datetime(2019, 9, 11, 14, 0),
+                datetime(2019, 9, 12, 13, 0),
+                datetime(2019, 9, 12, 13, 30),
+            ],
+        )
+        kwargs.setdefault(
+            'code', 'dummy#{}'.format(id_) if id_ is not None else u'dummy'
+        )
+        kwargs.setdefault(
+            'title', 'Test event {}'.format(id_) if id_ is not None else u'Test event'
+        )
+        kwargs.setdefault('creator_name', 'Dummy')
+        kwargs.setdefault('duration', timedelta(minutes=60))
+        kwargs.setdefault('private', True)
+        kwargs.setdefault('timezone', 'Europe/Zurich')
+        newdle = Newdle(
+            id=id_,
+            creator_uid=dummy_uid,
+            **kwargs,
+        )
+        db_session.add(newdle)
+        db_session.flush()
+        return newdle
+
+    return _create_newdle
+
+
+@pytest.fixture
+def override_config(app):
+    """Returns a callable which lets you override app config variables."""
+
+    def _override_config(**kwargs):
+        app.config.update(**kwargs)
```

Comment:
I think this persists after the test finished (or failed)...

this should avoid it (untested!):

```python
@pytest.fixture
def override_config(app):
    """Returns a callable which lets you override app config variables."""
    orig_config = dict(app.config)

    def _override_config(**kwargs):
        app.config.update(**kwargs)

    try:
        yield _override_config
    finally:
        app.config.update(orig_config)
```

Reply:
Yep, works like that. Tested it and:

Before:
```
 'FINAL_DATE_CLEANUP_INTERVAL': 30,
 'LAST_ACTIVITY_CLEANUP_INTERVAL': 90,
```

After:
```
 'FINAL_DATE_CLEANUP_INTERVAL': None,
 'LAST_ACTIVITY_CLEANUP_INTERVAL': None,
```